### DISCUSSION TITLE
请问这个 vue-mini 框架对比 uni-app 优劣如何呢？

### DISCUSSION THREAD

**Discussion by haro1d** [vue-mini/vue-mini#72] [General]
视频里，博主只是对比了taro，而没有对比更多人关注、更多vue使用者会首选的 uni-app

**Comment by mhpsy** [comment]
vue-mini 比较轻量，没有“魔法”，只是优化了写原生小程序逻辑侧的体验。
uniapp更大更重魔法很多，但是写vue语法就能够变成抖音、百度、快手等等各种小程序+安卓+IOS

**Comment by typed-sigterm** [comment]
https://vuemini.org/guide/comparisons.html#uni-app-mpvue-megalo

**Comment by yangmingshan** [comment]
我更新了文档的「比较」章节，其中包括跟 uni-app 的对比。
https://vuemini.org/guide/comparisons.html#uni-app
